week-01-controlling-the-dom/day-04-dom/dawn-jquery/README.md

```diff
@@ -0,0 +1,83 @@
+
+But some would argue, [you might not need jQuery](http://youmightnotneedjquery.com/)! It can be overkill, depending on what you're doing!
+
+#### How to get jQuery
+You can only use jQuery if it's included in your page.
+
+The quickest way to include jQuery in your project is to grab the jQuery library using a "CDN" (a blazing fast "content delivery network") and dropping it into a script tag in the head of your html (just google "jQuery cdn" and copy paste!):
+
+``` html
+<!DOCTYPE html>
+<html>
+    <head>
+        <script type="text/javascript" src="http://code.jquery.com/jquery-2.1.3.min.js"></script>
+    </head>
+    <body>
+        <!-- Nothing here yet! -->
+    </body>
+</html>
+
+```
+
+Sites like: github.com, css-tricks.com, and jquery.com (!) all include the jQuery library on their page. This means all you have to do is open up your Chrome Developer Console on one of those sites, and you can start playing with jQuery on the page!
+
+#### jQuery is just Javascript
+Everything you do in jQuery can be done in pure/vanilla javascript.
+
+Take a look at the [raw jquery library](http://code.jquery.com/jquery-2.1.3.js) (it's just a bunch of javascript!).
+
+#### Using jQuery
+You know you're working with jQuery any time you see a `$` in your code.
+
+As you're working with jQuery to manipulate DOM elements, you are almost always either *getting* or *setting* a value. Memorize this pattern:  
+- `$("CSS Selector").someJqueryMethodName()` - **getting** a value  
+- `$("CSS Selector").someJqueryMethodName(setterValue)` - **setting** a value
+
+For example, if you are viewing this readme on github.com, then you can try the following in your Chrome Developer Console:  
+- `$('article').text()` -- **get** the text of the readme (it lives inside of an `article` tag)  
+- `$('article').text("Boo!")` -- **set** the text of the readme to "Boo!"
+
+Wowa!
+
+Let's try another:  
+- `$('article').css("background-color")` -- **get** the background color  
+- `$('article').css("background-color", "blue")` -- **set** the background color to blue.
+
+#### Documentation
+Check out the `.text()` and `.css()` methods in the jQuery API Documentation: [text](http://api.jquery.com/text/), [css](http://api.jquery.com/css/).
+- Pay close attention in the documentation: there's one section that talks about how to "get" text, and there's another section that talks about how to "set" text.
+- `.text()` and `.css()` are not native javascript methods! They only work on jQuery-wrapped DOM Elements (that's what that `$` is doing).
+
+For more cool DOM manipulation tricks, you'll need to hit the docs:
+- jQuery API [Documentation]([jquery.com](jquery.com))
+    - [Manipulation Methods](http://api.jquery.com/category/manipulation/)
+    - [Tree Traversal Methods](http://api.jquery.com/category/traversing/)
+
+The more you struggle with this kind of documentation, the stronger your coding-chops will become!
+
+## Exercises
```

Comment:
- Link to exercises.md
- Link to github-hack
- Remove Street Fighter
- Keep CSS Diner